samples/boards/nxp/poweroff/boards/frdm_mcxa156.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		poweroff-wakeup-source = &lptmr0;
+	};
+};
+
+&lptmr0 {
+	status = "okay";
+	timer-mode-sel = <0>;
+	clk-source = <1>;
+	clock-frequency = <16384>;
+	resolution = <32>;
+	wakeup-source;
+};
```

Comment:
all these MCXA board overlay for this sample are the same, @hakehuang what is your opinion about how to do this one ? make one overlay for all of them? or it complicates the testcase yaml ? (but we already are adding a lot of cases anyways)